cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/OriginalDataAgent.scala

```diff
@@ -69,9 +69,10 @@ class OriginalDataAgent(val dataSetInfo: DataSetInfo,
     }
     val temporalSchema = dataSetInfo.schema.asInstanceOf[Schema]
     val now = DateTime.now().minusMillis(1)
-    val filter = FilterStatement(temporalSchema.timeField, None, Relation.inRange, Seq(start, now).map(TimeField.TimeFormat.print))
+    val startFilter = FilterStatement(temporalSchema.timeField, None, Relation.>, Seq(start).map(TimeField.TimeFormat.print))
```

Comment:
Sorry for the delay. Here we think that the original usage of `inRange` logical operator is good practice to maintain the same semantic meaning for different DB adapters no matter how the DB adapters  implement this operator.

Reply:
And after discussion with @dayuebai, we agreed that the problem is caused by the first cardinality counting query was using the >= and <= operators but later collecting states queries are using this `inRange` operator which is usually implement as >= and <. So that the intersect one day's records are counted twice. So we proposed the solution to see if we can modify the first query to also use this `inRange` operator.

Reply:
It's good to maintain the generality of the function.